app/main.py

```diff
@@ -1 +1,17 @@
 # write your code here
+def copy_file(command: str) -> None:
+    command_index = 0
+    source_file_index = 1
+    target_file_index = 2
+
+    if command.split()[command_index] != "cp":
+        return
+
+    source_file = command.split()[source_file_index]
+    target_file = command.split()[target_file_index]
```

Comment:
comand, source_file, target_file = command.split()

Reply:
great comment, though there are fewer lines the code looks much clearer. Thanks!